release/long_running_tests/workloads/many_drivers.py

```diff
@@ -78,6 +78,9 @@ def method(self):
         actor = Actor._remote(args=[], kwargs={{}}, resources={{str(i): 1}})
         assert ray.get(actor.method.remote()) == 1
 
+# Tests datasets doesn't leak workers.
+ray.data.range(100).map(lambda x: x).take()
```

Comment:
Is it possible to capture this in a unit test, or is it not worth it?

Reply:
It's a little tricky, so I punted on it for now.